panacea-nft/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "panacea-nft"
+version = "0.1.0"
+edition = "2018"
+description = "panacea-nft"
+repository = "https://github.com/gyuguen/panacea-smart-contract/panacea-nft"
+
+exclude = [
+    # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+    "artifacts/*",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+apt-level = 3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
+panic = 'abort'
+incremental = false
+overflow-checks = true
+
+[features]
+backtraces = ["cosmwasm-std/backtraces"]
+
+[dependencies]
+cw721 = "0.6.2"
+cw721-base = { version = "0.6.2", features = ["library"] }
+cw-storage-plus = "0.6.2"
+
+cosmwasm-std = "0.14.0"
+cosmwasm-storage = "0.14.0"
+schemars = "0.8.3"
+thiserror = "1.0.23"
+serde = { version = "1.0.126", default-features = false, features = ["derive"] }
+serde_json = "1.0.66"
+log = "0.4"
+rand = "0.8.4"
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
I found the cause of the problem.
There's something going on with this lib.
Attempted to change to a non-conflicting version, but an error occurs when the function in the rand lib is called.
I will remove the part that generates token_id with random, and use the auto-increment value.